addon/components/new-osf-navbar/style.scss

```diff
@@ -0,0 +1,228 @@
+// Overrides some of base osf-styles for the navbar in osf-style/sass/_bootstrap.scss
+$osf-teal-navbar: #2E6A74;
+$osf-highlight-navbar: #5A8288;
+$osf-dark-blue-navbar: #263947;
+
+.new-osf-navbar-styling {
+    /* Overrides the overall navbar color */
+    .navbar-inverse {
+        background-color: $osf-dark-blue-navbar;
+    }
+
+    /* Enlarges OSF Font */
+    .navbar-brand {
+        font-size: 28px;
+    }
+
+    /* Enlarges OSF logo */
+    .osf-navbar-logo {
+        width: 35px;
+        height: 35px;
+        margin-top: -4px;
+        margin-left: -4px;
+    }
+
+    /* Defines position of primary navigation: dropdown toggle (caret) */
+    .primary-nav {
+        float: left;
+        /* Overrides vertical and horizontal position of dropdown caret for service menu */
+        li.dropdown {
+            margin-left: -46px;
+            padding-top: 15px;
+            list-style-type: none;
+            font-size: 15px;
+        }
+    }
+
+    /* Overrides the color, relative position, and font size of dropdown menus (both service and auth) */
+    .dropdown-menu {
+        background-color: $osf-teal-navbar;
+        &.service-dropdown {
+            li > a {
+               font-size: 16px;
+            }
+            top: 58px;
+            left: -14.5px;
+        }
+        &.auth-dropdown {
+            li > a {
+               font-size: 15px;
+            }
+            top: 60px;
+            left: 15px;
+        }
+    }
+
+    /* Overrides background of dropdown-toggle when active - needed for auth navbar */
+     .navbar-nav > .open > a,
+     .navbar-nav > .open > a:focus,
+     .navbar-nav > .open > a:hover {
+         background: transparent;
+         box-shadow: none;
+     }
+
+     /* Overrides font-size and position of user name / gravatar */
+     a.nav-user-dropdown {
+         padding-right: 0;
+         font-size: 15px;
+     }
+
+    /* Overrides color of dropdown list hover (applies to both service and auth dropdowns) */
+    .dropdown-menu > li > a:hover,
+    .dropdown-menu > li > a:focus {
+          background-image: -webkit-linear-gradient(top, $osf-highlight-navbar 0%, $osf-highlight-navbar 100%);
```

Comment:
There's an [ember-cli addon](https://github.com/kimroen/ember-cli-autoprefixer) that helps with this. The benefit is that over time, we can eliminate the prefixes for unsupported browsers and not have to think about it.